packages/react-core/src/components/MenuToggle/MenuToggle.tsx

```diff
@@ -55,7 +55,8 @@ export class MenuToggleBase extends React.Component<MenuToggleProps> {
           styles.menuToggle,
           isExpanded && styles.modifiers.expanded,
           variant === 'primary' && styles.modifiers.primary,
-          isPlain && styles.modifiers.plain,
+          (isPlain || variant === 'plainText') && styles.modifiers.plain,
```

Comment:
Can you move the logic of checking the variant to line 39 where `isPlain` is set.